whois-commons/src/main/java/net/ripe/db/whois/common/grs/RipeAuthoritativeResourceImportTask.java

```diff
@@ -29,11 +31,17 @@
 
     private final String rsngBaseUrl;
     private final Client client;
+    private final boolean useSingleApi;
+    private final String apiKey;
+    private final ExecutorService executorService;
 
     @Autowired
     public RipeAuthoritativeResourceImportTask(final ResourceDataDao resourceDataDao,
                                                @Value("${grs.import.enabled:false}") final boolean enabled,
-                                               @Value("${rsng.base.url:}") final String rsngBaseUrl) {
+                                               @Value("${rsng.base.url:}") final String rsngBaseUrl,
```

Comment:
Can you add a class-level JavaDoc comment explaining how those properties work (as in, how to select each RSNG endpoint, and how to fall back to the nightly delegated-stats dump).

Reply:
There is nothing that needs to be done apart from setting the flag value to true or false. 
Or do you want me to just add a javadoc explaining what the flag is about?

Reply:
OK, leave it for now, we can document it elsewhere (the GRS import involves a few classes, not just this one)